src/com/company/HW_less_2.java

```diff
@@ -0,0 +1,57 @@
+package com.company;
+
+public class HW_less_2 {
+
+    public static void main(String[] args) {
+
+        gapNum(5,61);
+        numType(-6);
+        numType_2(0);
+        print("Слово которое печатать ", 4);
+        checkLeapYear(2130);
+
+    }
+
+    public static void gapNum(int a, int b){
+        int sum = a+b;
+        // можно конечно и без объявление переменной sum, но сделал так :)
+
+        // if(sum >= 10 && sum <= 20) System.out.println("true");
+        // else System.out.println("false");
+
+        System.out.println(sum >= 10 && sum <= 20);
```

Comment:
метод лучше сделать возвращаемым boolean, а в консоль уже выводить на вызывающей стороне (в методе main)
```suggestion
        return sum >= 10 && sum <= 20;
```